variable_set_test.go

```diff
@@ -190,6 +190,67 @@ func TestVariableSetsDelete(t *testing.T) {
 	})
 }
 
+func TestVariableSetsApplyToAndRemoveFromWorkspaces(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgTest, orgTestCleanup := createOrganization(t, client)
+	defer orgTestCleanup()
+
+	vsTest, _ := createVariableSet(t, client, orgTest, VariableSetCreateOptions{})
```

Comment:
While I know deleting an organization will cascade and delete any children relationships, there have been instances where this cascade behavior fails. Therefore, I'd still `defer` each cleanup method to be on the safe side. This one is not a huge deal and is very edge case, so feel free to `defer` this comment (😆).

Reply:
My particular discipline is figuring out why impossible things keep happening, so I'm happy to add a `defer` here. :D

Reply:
Addressed!